uaclient/entitlements/fips.py

```diff
@@ -62,15 +63,27 @@ def application_status(self) -> "Tuple[status.ApplicationStatus, str]":
             "Reboot to FIPS kernel required",
         )
 
-    def disable(self, silent: bool = False) -> bool:
-        """FIPS cannot be disabled, so simply display a message to the user"""
-        if not silent:
-            print("Warning: no option to disable {}".format(self.title))
-        return False
-
-    def _cleanup(self) -> None:
-        """FIPS can't be cleaned up automatically, so don't do anything"""
-        pass
+    def remove_packages(self) -> None:
```

Comment:
Why are we removing FIPS packages on disable ? If the problem is the package holds, shouldn't we just unhold them on disable ?

Reply:
We are only removing `ubuntu-fips` meta package in this operation, the meta-package (on removal) will unset the grub config options which essentially deactivates the fips -hmac packages so the pkg needs to be removed in this case. It is not an operation that we should be doing for any other entitlement.